src/index.js

```diff
@@ -133,7 +133,7 @@ class CID {
     /**
      * @type {string}
      */
-    this.multibaseName = multibaseName
+    this.multibaseName = multibaseName || (version === 0 ? 'base58btc' : 'base32')
```

Comment:
Should this throw an error if version is 0 and `multibaseName` is not `base58btc`?

Reply:
that's handled by the next bit, `CID.validateCID(this)` which does this in cid-util.js#CIDUtil.checkCIDComponents():

```js
    if (other.version === 0) {
      if (other.codec !== 'dag-pb') {
        return `codec must be 'dag-pb' for CIDv0`
      }
      if (other.multibaseName !== 'base58btc') {
        return `multibaseName must be 'base58btc' for CIDv0`
      }
    }
```

That results in a `throw` with that message in `validateCID()` down below.